contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreeView/Diff/index.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+
+  <span :class="colorClass">
+    <slot :sign="sign" :value="Math.abs(value)"></slot>
+  </span>
+
+</template>
+
+<script>
+
+  export default {
+    name: 'Diff',
+    props: {
+      value: {
+        type: Number,
+        required: true,
+      },
+    },
+    computed: {
+      colorClass() {
+        if (this.value < 0) return 'red--text text--darken-1';
+        if (this.value > 0) return 'green--text text--darken-1';
+        return 'grey--text text--darken-1';
+      },
+      sign() {
+        if (this.value > 0) {
+          return '+ ';
```

Comment:
Does the positive/negative sign vary across cultures? Not sure if it would make sense to create strings to wrap the text like
```
$trs: {
  positiveText: '+ {text}',
  negativeText: '- {text}',
}
```

Reply:
That's a good point. I haven't been able to find out yet, but wrapping it into a translation string seems to be much safer to me and won't harm anything in any case.